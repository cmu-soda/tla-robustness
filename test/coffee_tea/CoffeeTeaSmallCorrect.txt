Spec is robust against ANY behavior or environment
