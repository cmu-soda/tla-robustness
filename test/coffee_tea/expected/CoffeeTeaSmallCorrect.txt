TLA+ Module: CoffeeTeaSmallCorrect
Diff rep grouped by action:
CoffeeTeaSmallCorrect is robust against ANY behavior or environment
